Modules/Loadable/Segmentations/qSlicerSegmentationsNodeWriter.cxx

```diff
@@ -45,10 +45,9 @@
 class qSlicerSegmentationsNodeWriterPrivate
 {
 public:
-  QString Description;
+  QString Description; // These member variables don't get used anywhere.
```

Comment:
If you are sure they are not used anywhere then please remove them.

Reply:
I think the class `qSlicerSegmentationsNodeWriterPrivate` is not getting used anywhere. Should I remove it?

Reply:
I removed it